src/utils/query.js

```diff
@@ -187,6 +187,44 @@ export function getQueryName(meta) {
   return basePath;
 }
 
+/**
+ * Create query name based on query settings for use as object keys (used
+ * in listener management and reducers).
+ * @param  {Object} meta - Metadata object containing query settings
+ * @param  {String} meta.collection - Collection name of query
+ * @param  {String} meta.doc - Document id of query
+ * @param  {Array} meta.subcollections - Subcollections of query
+ * @return {String} String representing query settings
+ */
+export function getBaseQueryName(meta) {
```

Comment:
Rather than add an optional parameter to the existing `getQueryName` function, I opted to simply copy the logic here and remove the addition of the document id.   Since it's pretty much identical, it may be a good idea at some point to have a (non exported) third method that both will call into.